.env

```diff
@@ -0,0 +1 @@
+MONGO_URL="mongodb+srv://polkaPetun:0sD0oW813Ik0yMXL@cluster0.wjjmc.mongodb.net/happyThoughts?retryWrites=true&w=majority"
```

Comment:
We're assuming that this is the old connection string from before you listed .env in .gitignore. It's probably still better to remove it.